contrib/dev-docker/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -efuxo pipefail
+
+# cd to the repo root:
+cd "$(readlink -f "$0/../../../")"
+docker build \
+    -t zcash-dev-docker \
+    --build-arg "BUILDER_UID=$(id -u)" \
```

Comment:
On linux, this will need to be run with sudo (as docker doesn't allow execution without superuser permissions.); however, if you run the whole `run.sh` script with sudo then `id -u` returns 0, which then fails the `adduser` call in the dockerfile.

This should be documented and overrideable.